src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/QuotableEncoder.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.utils
+
+import kotlin.random.Random
+
+class QuotableEncoder(private val delimiter: String? = "-", private val minLength: Int? = 1, private val chunkSize: Int? = 2) {
```

Comment:
could these be non nullable?  Always a default rather than null?

Reply:
Good spot, will update.